packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -296,4 +299,16 @@ class UserPreferences extends ChangeNotifier {
     await _sharedPreferences.setString(_TAG_USER_PICTURE_SOURCE, source.tag);
     notifyListeners();
   }
+
+  SmoothBarcodeScannerType scanningEngine() {
+    final String name =
+        _sharedPreferences.getString(_DEV_MODE_SCANNING_ENGINE) ?? 'mlkit';
```

Comment:
Does it work? I thought we always had the enum prefix first, like `SmoothBarcodeScannerType.mlkit`?
Or is it because I lazily use `.toString()` and you use `.name`?

Reply:
Tbh I didn't check that beforehand, just tested and it works. But indeed 

``` dart
print(SmoothBarcodeScannerType.mlkit.name); => mlkit
print(SmoothBarcodeScannerType.mlkit.toString()); => SmoothBarcodeScannerType.mlkit
```